src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsprobationestateapi/integration/regions/GetAllRegions.kt

```diff
@@ -1,20 +1,22 @@
 package uk.gov.justice.digital.hmpps.hmppsprobationestateapi.integration.regions
 
+import kotlinx.coroutines.flow.toList
+import kotlinx.coroutines.runBlocking
 import org.junit.jupiter.api.Test
 import uk.gov.justice.digital.hmpps.hmppsprobationestateapi.db.entities.Region
 import uk.gov.justice.digital.hmpps.hmppsprobationestateapi.integration.IntegrationTestBase
 
 class GetAllRegions : IntegrationTestBase() {
 
   @Test
-  fun `must get all regions`() {
+  fun `must get all regions`(): Unit = runBlocking {
     val regions = regionRepository.saveAll(
       listOf(
         Region(code = "REGION1", name = "First Region", new = true),
         Region(code = "REGION2", name = "Second Region", new = true),
         Region(code = "REGION3", name = "Third Region", new = true)
       )
-    ).collectList().block()!!
+    ).toList(ArrayList())
```

Comment:
I don't suppose there is a .await() or similar here? Might be clearer what the intention is.